src/Form/AutologoutSettingsForm.php

```diff
@@ -200,20 +200,26 @@ public function buildForm(array $form, FormStateInterface $form_state) {
    */
   public function validateForm(array &$form, FormStateInterface $form_state) {
     $values = $form_state->getValues();
+    $new_stack = [];
+    foreach($values['table'] as $key => $pair) {
+      if(is_array($pair)) {
+        foreach($pair as $pairkey => $pairvalue) {
+          $new_stack[$pairkey] = $pairvalue;
+        }
+      }
+    }
     $max_timeout = $values['autologout_max_timeout'];
-
     // Validate timeouts for each role.
     foreach (user_roles(TRUE) as $key => $role) {
-      if (empty($values['autologout_role_' . $key])) {
+      if (empty($new_stack['role_' . $key])) {
         // Don't validate role timeouts for non enabled roles.
         continue;
       }
 
-      $timeout = $values['autologout_role_' . $key . '_timeout'];
+      $timeout = $new_stack['role_' . $key . '_timeout'];
       $validate = autologout_timeout_validate($timeout, $max_timeout);
-
       if (!$validate) {
-        $form_state->setErrorByName('role_' . $key . '_timeout', $this->t('%role role timeout must be an integer greater than 60, less then %max or 0 to disable autologout for that role.', array('%role' => $role, '%max' => $max_timeout)));
+        $form_state->setErrorByName('role_' . $key . '_timeout', $this->t('!role role timeout must be an integer greater than 60, less then !max or 0 to disable autologout for that role.', array('!role' => $key, '!max' => $max_timeout)));
```

Comment:
I will revert this change. But I do not get any fatal error on installation/uninstallation.